docs/_themes/.gitignore

```diff
@@ -0,0 +1,3 @@
+*.pyc
```

Comment:
This is fine, but these ignores are already included in the base directory's `.gitignore` file.